### PR TITLE
Pfw 873

### PR DESCRIPTION
**Pull request by MRprusa3d**

### CODE REVIEW

**Review by DRracer — COMMENTED** [review]
Code size reduction of this PR is desperately needed. For now I propose to merge this into master, but we must return to this PR

File: Firmware/Configuration.c [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#include <stdint.h>
+#include <avr/pgmspace.h>
+//!?! #include "Configuration.h"
+#include "Configuration_prusa.h"
+
+const uint16_t _nPrinterType PROGMEM=PRINTER_TYPE;
+const char _sPrinterName[] PROGMEM=PRINTER_NAME;
+const uint16_t _nPrinterMmuType PROGMEM=PRINTER_MMU_TYPE;
+const char _sPrinterMmuName[] PROGMEM=PRINTER_MMU_NAME;
+
```
Comment (DRracer) [inline Firmware/Configuration.c]: don't use a C file when not necessary. Rename to Configuration.cpp and you won't have problems linking these variables without "extern "C"".

File: Firmware/mmu.cpp [hunk 2]
```diff
@@ -265,6 +267,9 @@ void mmu_loop(void)
 			FDEBUG_PRINTF_P(PSTR("MMU => '%dok'\n"), mmu_finda);
 			puts_P(PSTR("MMU - ENABLED"));
 			mmu_enabled = true;
+//-//
+// ... PrinterType/Name
+fSetMmuMode(true);
```
Comment (mkbel) [inline Firmware/mmu.cpp]: Missing indentation.

File: Firmware/printers.h [hunk 3]
```diff
@@ -3,14 +3,49 @@
...
 #define PRINTER_MK25_SNMM   251
-#define PRINTER_MK25S		252
+#define PRINTER_MK25S	   252
 #define PRINTER_MK3         300
 #define PRINTER_MK3_SNMM    301
-#define PRINTER_MK3S		302
+#define PRINTER_MK3S	   302
+*/
+// *** MK1
+#define PRINTER_MK1	               100
```
Comment (mkbel) [inline Firmware/printers.h:7]: Same macro defined twice.

File: Firmware/printers.h [hunk 4]
```diff
@@ -3,14 +3,49 @@
...
+#define PRINTER_MK1_NAME           "MK1"
+// *** MK2
+#define PRINTER_MK2	               200
```
Comment (mkbel) [inline Firmware/printers.h:10]: Same macro defined twice.

File: Firmware/printers.h [hunk 5]
```diff
@@ -3,14 +3,49 @@
...
+#define PRINTER_MK2_NAME           "MK2"
+#define PRINTER_MK2_SNMM           201            // better is "10200"
```
Comment (mkbel) [inline Firmware/printers.h:12]: Same macro defined twice.

**Review by mkbel — CHANGES_REQUESTED** [review]

**Review by mkbel — APPROVED** [review]

**Pull request merged**